Audiobook Player/Player/Containers/PlayerControlsViewController.swift

```diff
@@ -8,69 +8,225 @@
 
 import UIKit
 
-class PlayerControlsViewController: PlayerContainerViewController {
+class PlayerControlsViewController: PlayerContainerViewController, UIGestureRecognizerDelegate {
+    @IBOutlet private weak var artworkView: UIView!
     @IBOutlet private weak var artwork: UIImageView!
     @IBOutlet private weak var playPauseButton: UIButton!
-    @IBOutlet private weak var rewindButton: UIButton!
-    @IBOutlet private weak var forwardButton: UIButton!
+    @IBOutlet private weak var rewindIcon: PlayerForwardIconView!
```

Comment:
I think `rewind`and `forward` references are swapped in the storyboard, right now the player controls are showing like this `-30` `play/pause` `+30`

Reply:
updated comment

Reply:
🤦‍♂️ they are